### PR TITLE
OSOE-56: Upgrade to 1.4.0

### PR DESCRIPTION
**Pull request by barthamark**
FOLK-190
OSOE-56

### CODE REVIEW

**Review by DemeSzabolcs — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,7 @@
 obj/
 bin/
 *.user
-wwwroot/
-node_modules/
\ No newline at end of file
+# Temporarily adding wwwroot to the version control.
+# wwwroot/
+node_modules/
+.idea
```
Comment (DemeSzabolcs) [inline .gitignore]: Why did you add these?

File: Lombiq.ContentEditors.Samples/Lombiq.ContentEditors.Samples.csproj [hunk 2]
```diff
@@ -1,31 +1,48 @@
...
-
-    <ItemGroup>
-        <PackageReference Include="OrchardCore.Module.Targets" Version="1.3.0" />
-        <PackageReference Include="OrchardCore.Contents" Version="1.3.0" />
-    </ItemGroup>
-
-    <ItemGroup>
-      <ProjectReference Include="..\Lombiq.ContentEditors\Lombiq.ContentEditors.csproj" />
-    </ItemGroup>
+  <PropertyGroup>
```
Comment (DemeSzabolcs) [inline Lombiq.ContentEditors.Samples/Lombiq.ContentEditors.Samples.csproj:3]: So the README and the README for the Sample project (and other stuff needed for open sourcing) will be done in https://lombiq.atlassian.net/browse/OSOE-56 right? (I'm just commenting on this line randomly)

**Review by DemeSzabolcs — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (barthamark) [reply .gitignore]: .idea is Rider's .vs folder that must be ignored, but you are right, it can be on the superrepository-level.

File: Lombiq.ContentEditors.Samples/Lombiq.ContentEditors.Samples.csproj (hunk 2 above)
Comment (barthamark) [reply Lombiq.ContentEditors.Samples/Lombiq.ContentEditors.Samples.csproj:3]: Yes, that's correct. For now the goal is to have it in a separate repository and have it somewhat functional.

**Review by DemeSzabolcs — APPROVED** [review]
